grid_map_pcl/test/GridMapPclLoaderTest.cpp

```diff
@@ -6,89 +6,110 @@
  *      Institute: ETH Zurich, Robotic Systems Lab
  */
 
-#include <cstdlib>
 #include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
+#include <rcutils/error_handling.h>
 
-#include "grid_map_pcl/GridMapPclLoader.hpp"
+#include <cstdlib>
+#include <memory>
+#include <string>
 
+#include "grid_map_pcl/GridMapPclLoader.hpp"
 #include "PointcloudCreator.hpp"
 #include "test_helpers.hpp"
 
-namespace grid_map {
-namespace grid_map_pcl_test {
-
-TEST(GridMapPclLoaderTest, FlatGroundRealDataset)
+namespace grid_map
+{
+namespace grid_map_pcl_test
 {
-  grid_map_pcl_test::setVerbosityLevel(ros::console::levels::Warn);
+class GridMapPclLoaderTest : public ::testing::Test
+{
+protected:
+  void SetUp() override
+  {
+    node = std::make_shared<rclcpp::Node>("GridMapPclLoaderTest");
+
+    // Changing the logging verbosity to WARN
```

Comment:
I think you mean `DEBUG` ?

Reply:
I think originally the logging level was being changed to `WARN`, so I think the comment is correct, but I messed up the actual change of the logging level so fixed it now to`WARN`. Does that sound right?

Reply:
Yes.